qiskit_machine_learning/neural_networks/__init__.py

```diff
@@ -39,10 +43,8 @@
    :toctree: ../stubs/
    :nosignatures:
 
-   NeuralNetwork
    OpflowQNN
    TwoLayerQNN
-   SamplingNeuralNetwork
```

Comment:
Do you think we don't need base classes in the documentation? I don't a strong opinion, just curious.

Reply:
If you look at the `Neural Network Base Classes` section just above `NeuralNetwork` was there too. So I just removed the duplicate from `Neural Networks` section since having it twice seemed like a mistake. Since `SamplingNeuralNetwork` is and abstract class too I moved it up in the base classes section so the concrete instances were separate.

In answer to your specific question they should be there. We might expect people to extend them themselves, or want to refer to methods etc of the base class. Anything we consider public ought to be in the API ref,

Reply:
Got it. Thanks!